src/components/Navbar/ThemePicker.tsx

```diff
@@ -1,46 +1,99 @@
 import Moon from 'components/icons/Moon'
 import Sun from 'components/icons/Sun'
-
 import { Trans } from '@lingui/macro'
 
 import { ThemeContext } from 'contexts/themeContext'
-import React, { useContext } from 'react'
+import React, { CSSProperties, useContext } from 'react'
 
 import { ThemeOption } from 'constants/theme/theme-option'
 
 export default function ThemePicker({ mobile }: { mobile?: boolean }) {
-  const { themeOption, setThemeOption } = useContext(ThemeContext)
+  const {
+    themeOption,
+    setThemeOption,
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  const iconSize = 18
+  const padding = 6
+  const height = iconSize + padding * 2
+  const selectedColor = colors.icon.primary
+  const unselectedColor = colors.icon.tertiary
+
+  const iconStyle: CSSProperties = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingTop: padding,
+    paddingBottom: padding,
+  }
 
-  const iconSize = 16
+  const switchStyle: CSSProperties = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    cursor: 'pointer',
+    width: iconSize * 2 + padding * 4,
+    marginRight: 38,
+    height,
+    borderRadius: height / 2,
+  }
 
+  if (!mobile) {
+    return (
+      <div
```

Comment:
**suggestion**: would probably make sense to have a mobile and desktop specific component, to reduce complexity of this template. Not big deal though